chapters/usage-profile.md

```diff
@@ -0,0 +1,326 @@
+It should recorded from build system tools or date field of the package file.
+
+### 31.5.3 Examples
+
+EXAMPLE 1 Tag: `PackageBuiltDate:`
+
+```text
+PackageBuiltDate: 2010-01-29T18:30:22Z
+```
+
+EXAMPLE 2 RDF: Property `spdx:packageBuiltDate` in class `spdx:UsageScope`
+
+```text
+<UsageScope rdf:about="...">
+    <packageBuiltDate> 2010-01-29T18:30:22Z </packageBuiltDate>
+</UsageScope>
+```
+
+## 31.6 Valid Until Date field <a name="31.6"></a>
+
+### 31.6.1 Description
+
+Identify expiration date of designated "usage scope" descriptions which declared by supplier of the package. 
+
+Table 31.6 — Metadata for the valid until date
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..1 |
+| Format | `YYYY-MM-DDThh:mm:ssZ`<br>where:<br><ul><li>`YYYY` is year</li><li>`MM` is month with leading zero</li><li>`DD` is day with leading zero</li><li>`T` is delimiter for time</li><li>`hh` is hours with leading zero in 24 hour time</li><li>`mm` is minutes with leading zero</li><li>`ss` is seconds with leading zero</li><li>`Z` is universal time indicator</li></ul> |
+
+
+### 31.6.2 Intent
+
+Description of the expiration date for specific usage scope.
+
+### 31.6.3 Examples
+
+EXAMPLE 1 Tag: `ValidUntilDate:`
+
+```text
+ValidUntilDate: 2030-01-29T18:30:22Z
+```
+
+EXAMPLE 2 RDF: Property `spdx:validUntilDate` in class `spdx:UsageScope`
+
+```text
+<UsageScope rdf:about="...">
+    <validUntilDate> 2030-01-29T18:30:22Z </validUntilDate>
+</UsageScope>
+```
+
+## 31.7 Valid Until Event field <a name="31.7"></a>
+
+### 31.7.1 Description
+
+Identify expiration condition of the usage profile descriptions or the package itself which declared by supplier of the package.
+For example, to specify expiration conditions of "usage scope" descriptions such as "These SPDX descriptions are valid until next delivery of newer SPDX descriptions". 
+In case, there are two or more "valid until" descriptions exist, designated "usage scope" description is expired when it meets an earliest condition defined in each "valid until" descriptions.
+
+Table 31.7 — Metadata for the valid until event field
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..* |
+| Format | Free form text that may span multiple lines, refers only to the immediately preceding "usage scope".  |
+
+
+### 31.7.2 Intent
+
+To the place for describing conditions / events which affects to validity of usage scope descriptions.
+
+### 31.7.3 Examples
+
+EXAMPLE 1 Tag: `ValidUntilEvent:`
+
+```text
+ValidUntilEvent: This Usage Scope descriptions are valid until updated descriptions are delivered
+```
+
+
+EXAMPLE 2 RDF: Property `rdfs:comment` in class `spdx:validUntilEvent` of `spdx:UsageScope`
+
+```text
+<UsageScope rdf:about="...">
+    <validUntilEvent>
+         <rdfs:comment> Updated descriptions are delivered</rdfs:comment>
+    </validUntilEvent>
+</UsageScope>
+```
+
+## 31.8 ValidUntil SPDX identifier field <a name="31.8"></a>
```

Comment:
Same question here as for UsageScopeSPDXID above -- I'm not sure I understand the purpose of this being a separate field; isn't this duplicative of having the SPDX ID in the UsageScope field above?